src/scene/materials/standard-material.js

```diff
@@ -324,6 +324,20 @@ class Chunks {
  *
  * // Notify the material that it has been modified
  * material.update();
+ * @example
+ * // Create a new Standard material
+ * var material = new pc.StandardMaterial();
+ *
+ * // assign a texture to diffuse slot
+ * material.diffuseMap = texture;
+ *
+ * // use the alpha channel of the texture for alpha testing with a reference value of 0.5
+ * material.opacityMap = texture;
+ * material.opacityMapChannel = "a";
```

Comment:
[This line](https://github.com/playcanvas/engine/blob/master/src/scene/materials/standard-material.js#L1134) seems to indicate that this _should_ be the default. Although perhaps there's a bug.

Reply:
Ideally, the default should be documented [here](https://github.com/playcanvas/engine/blob/master/src/scene/materials/standard-material.js#L167).

Reply:
Confirmed:

![image](https://user-images.githubusercontent.com/697563/108770329-13823300-7552-11eb-83d0-e7fe4915b31a.png)